runtime/kusama/src/lib.rs

```diff
@@ -1960,6 +1960,14 @@ sp_api::impl_runtime_apis! {
 	}
 
 	impl pallet_staking_runtime_api::StakingApi<Block, Balance> for Runtime {
+		fn inflation_rate() -> Perquintill {
+			let ideal_stake = 0_750_000;
+			let falloff = 0_050_000;
```

Comment:
I did not see any `ideal_staked` or `falloff` here in the Kusama runtime. Confirmation of these values being correct is needed.